pkgs/applications/gis/saga/default.nix

```diff
@@ -77,9 +74,11 @@ mkDerivation rec {
     sqlite
   ];
 
-  enableParallelBuilding = true;
+  cmakeFlags = [
+    "-DOpenMP_SUPPORT=${if stdenv.isDarwin then "OFF" else "ON"}"
+  ];
 
-  CXXFLAGS = lib.optionalString stdenv.cc.isClang "-std=c++11 -Wno-narrowing";
+  enableParallelBuilding = true;
```

Comment:
isn't this the default with cmake? I might be wrong about this.

Reply:
Absolutely. Thanks!